packages/components/src/components/Warning/Warning.tsx

```diff
@@ -79,6 +80,20 @@ export const Warning = ({
 
     const withIcon = icon !== undefined;
 
+    const isMobile = useMediaQuery(`(max-width: ${SCREEN_SIZE.SM})`);
```

Comment:
argument of this function is going to be refactored in the future:
- we need to unify all responsive queries
- we need to take care of container queries, not just media queries